File: GremsReliquary/src/main/java/gremsReliquary/relics/cursed/DiabolicDiabola.java
package gremsReliquary.relics.cursed;

import com.badlogic.gdx.graphics.Texture;
import com.evacipated.cardcrawl.mod.stslib.relics.SuperRareRelic;
import com.megacrit.cardcrawl.core.AbstractCreature;
import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
import gremsReliquary.GremsReliquary;
import gremsReliquary.util.TextureLoader;
import kotlinReliquary.relics.AbstractGremRelic;

public class DiabolicDiabola extends AbstractGremRelic implements SuperRareRelic {
    public static final String ID = GremsReliquary.makeID(DiabolicDiabola.class.getSimpleName());
    public static final Texture IMG = TextureLoader.getTexture("gremsReliquaryResources/images/relics/DiabolicDiabola.png");
    public static final Texture OUTLINE = TextureLoader.getTexture("gremsReliquaryResources/images/relics/outline/DiabolicDiabola.png");
    public static int amount = 2;
    
    AbstractCreature p = AbstractDungeon.player;
    
    public DiabolicDiabola() {
        super(ID, IMG, OUTLINE, RelicTier.COMMON, RelicType.CURSED, LandingSound.MAGICAL);
    }
    
    // Description
    @Override
    public String getUpdatedDescription() {
        return DESCRIPTIONS[0];
    }
}

File: GremsReliquary/src/main/kotlin/kotlinReliquary/relics/cursed/DiabolicDiabola.kt
package kotlinReliquary.relics.cursed

import com.evacipated.cardcrawl.mod.stslib.relics.SuperRareRelic
import com.megacrit.cardcrawl.relics.AbstractRelic
import gremsReliquary.GremsReliquary
import gremsReliquary.util.TextureLoader
import kotlinReliquary.relics.AbstractGremRelic

class DiabolicDiabola : AbstractGremRelic(ID, IMG, OUTLINE, AbstractRelic.RelicTier.COMMON, AbstractGremRelic.RelicType.CURSED, AbstractRelic.LandingSound.MAGICAL), SuperRareRelic {
    companion object {
        val ID = GremsReliquary.makeID(DiabolicDiabola::class.java.simpleName)
        val IMG = TextureLoader.getTexture("gremsReliquaryResources/images/relics/DiabolicDiabola.png")
        val OUTLINE = TextureLoader.getTexture("gremsReliquaryResources/images/relics/outline/DiabolicDiabola.png")
        var amount = 2
    }

    override fun getUpdatedDescription(): String {
        return DESCRIPTIONS[0]
    }
}
